ospfd/ospf_vty.c

```diff
@@ -534,7 +534,7 @@ DEFUN (no_ospf_passive_interface,
 	}
 
 	if (ospf->vrf_id != VRF_UNKNOWN)
-		ifp = if_get_by_name(argv[1]->arg, ospf->vrf_id, 0);
+		ifp = if_get_by_name(argv[2]->arg, ospf->vrf_id, 0);
 
 	if (ifp == NULL) {
 		vty_out(vty, "interface %s not found.\n",
```

Comment:
The line directly below this line (541) needs to be fixed as well as that it will not display the interface name we think we are failing on.

Reply:
thanks!